apps/dapp/src/components/Metrics/Metrics.tsx

```diff
@@ -25,21 +25,16 @@ const Metrics = ({ treasuryMetrics, isHome, alignCenter }: MetricsProps) => {
   return (
     <Wrapper>
       <ApyWrapper>
-        <Apy
-          cryptoName={'$TEMPLE'}
-          value={`$${formatNumber(templeValue)}`}
-          isHome={isHome}
-          alignCenter={alignCenter}
-        />
+        <Apy cryptoName={'$TEMPLE'} value={`$${formatNumber(templeValue)}`} isHome={isHome} alignCenter={alignCenter} />
       </ApyWrapper>
-      <ApyWrapper>
+      {/* <ApyWrapper>
```

Comment:
Can't we just remove it? Having commented blocks of code just muddy things up.

Reply:
yeah fair enough, was thinking we might bring it back but I will just delete instead